astropy/timeseries/periodograms/lombscargle/implementations/scipy_impl.py

```diff
@@ -56,17 +66,30 @@ def lombscargle_scipy(t, y, frequency, normalization="standard", center_data=Tru
     if center_data:
         y = y - y.mean()
 
+    if SCIPY_LT_1_15:
+        kwargs = {}
+    else:
+        kwargs = {"floating_mean": fit_mean}
+
     # Note: scipy input accepts angular frequencies
-    p = signal.lombscargle(t, y, 2 * np.pi * frequency)
+    p = signal.lombscargle(t, y, 2 * np.pi * frequency, **kwargs)
+
+    if normalization not in ("psd", "standard", "log", "model", "psd"):
+        raise ValueError(f"{normalization=!r} not recognized")
 
     if normalization == "psd":
-        pass
-    elif normalization == "standard":
-        p *= 2 / (t.size * np.mean(y**2))
+        return p
+
+    if kwargs.get("floating_mean", False):
+        yoffset = y.mean()
```

Comment:
This is unnecessary if `center_data = True`.

Rather than parameterizing via `yoffset`, I think we should parameterize by `chi2_ref`, and take into account `center_data` for efficiency. Maybe something like this?
```python
if center_data or not fit_mean:
  chi2_ref = 0.5 * t.size * np.mean(y ** 2)
else:
  chi2_ref = 0.5 * t.size * np.mean((y - y.mean()) ** 2)
```